ocaml/xapi/xapi_clustering.ml

```diff
@@ -326,6 +326,21 @@ let rpc ~__context =
             "Can only communicate with xapi-clusterd through message-switch"
       )
 
+let maybe_switch_cluster_stack_version ~__context ~self ~cluster_stack =
+  if Xapi_cluster_helpers.corosync3_enabled ~__context then
+    let result =
+      Cluster_client.LocalClient.switch_cluster_stack (rpc ~__context)
+        "maybe_switch_cluster_stack_version" cluster_stack
```

Comment:
Can we use
```
let dbg = Context.string_of_task_and_tracing __context in
```
and `dbg` for that debuginfo parameter?

Reply:
Yep!